app.js

```diff
@@ -212,15 +212,16 @@ const getNodeConstants = () => new Promise((success, error) => {
 			return error({ success: false, error: err.message });
 		} else if (response.statusCode === 200) {
 			if (body && body.data) {
-				app.set('nodeConstants', body.data);
-				return success({
+				const results = {
 					version: body.data.version,
 					epoch: new Date(body.data.epoch) / 1000,
 					protocolVersion: body.data.protocolVersion,
-					nethash: body.data.nethash,
+					nethash: body.data.nethash || body.data.networkId,
```

Comment:
I think this is bit confusing, but is this ok?
the nethash cannot be used directly to sign transaction, but networkId can.
Maybe calling it networkId is better?

Reply:
It would be, the point is the Explorer does not interact with the network and the whole UI refers to that id.
The only other option is to pass the `networkId` through the backend and replace **all** references in the UI (there are many).
Does not make sense in my opinion, esp. that the project is being discontinued. And without specific logic it would break the v2.x compatibility.